modules/test-ui/src/App.tsx

```diff
@@ -1,69 +1,62 @@
 import { BrowserNode } from "@connext/vector-browser-node";
 import {
   getPublicKeyFromPublicIdentifier,
-  decrypt,
   encrypt,
-  ChannelSigner,
   createlockHash,
   getBalanceForAssetId,
   getRandomBytes32,
+  delay,
 } from "@connext/vector-utils";
 import React, { useEffect, useState } from "react";
 import pino from "pino";
-import { Wallet, constants } from "ethers";
-import { Col, Divider, Row, Statistic, Input, Typography, Table, Form, Button, Select, List, Collapse } from "antd";
+import { constants, utils } from "ethers";
+import { Col, Divider, Row, Statistic, Input, Typography, Table, Form, Button, List, Switch } from "antd";
+import { EngineEvents, EngineParams, FullChannelState, TransferNames } from "@connext/vector-types";
 
 import "./App.css";
-import { EngineEvents, FullChannelState, TransferNames } from "@connext/vector-types";
-
-import { config } from "./config";
-
-const logger = pino();
-
-const storedMnemonic = localStorage.getItem("mnemonic");
 
 function App() {
   const [node, setNode] = useState<BrowserNode>();
   const [channel, setChannel] = useState<FullChannelState>();
-  const [mnemonic, setMnemonic] = useState<string>();
 
   const [setupLoading, setSetupLoading] = useState<boolean>(false);
   const [connectLoading, setConnectLoading] = useState<boolean>(false);
   const [depositLoading, setDepositLoading] = useState<boolean>(false);
   const [requestCollateralLoading, setRequestCollateralLoading] = useState<boolean>(false);
   const [transferLoading, setTransferLoading] = useState<boolean>(false);
   const [withdrawLoading, setWithdrawLoading] = useState<boolean>(false);
+  const [entropy, setEntropy] = useState<string>("");
+  const [iframeSrc, setIframeSrc] = useState<string>("");
+  const [useRandomEntropy, setUseRandomEntropy] = useState<boolean>(true);
 
   const [connectError, setConnectError] = useState<string>();
 
   const [withdrawForm] = Form.useForm();
   const [transferForm] = Form.useForm();
 
   useEffect(() => {
-    const init = async () => {
-      if (!storedMnemonic) {
-        return;
+    const effect = async () => {
+      const storedEntropy = localStorage.getItem("entropy");
+      if (storedEntropy) {
+        setUseRandomEntropy(false);
       }
-      console.log("Found stored mnemonic, hydrating node");
-      await connectNode(storedMnemonic);
+      const storedIframeSrc = localStorage.getItem("iframeSrc");
+      setEntropy(storedEntropy);
+      setIframeSrc(storedIframeSrc || "http://localhost:3030");
     };
-    init();
+    effect();
   }, []);
 
-  const connectNode = async (mnemonic: string) => {
-    console.log("creating node with config", config);
+  const connectNode = async (iframeSrc: string, entropy: string): Promise<BrowserNode> => {
+    if (!iframeSrc) {
+      iframeSrc = "http://localhost:3030";
+    }
```

Comment:
Shouldn't this be derived from the config?

Reply:
The app (test UI is basically an example of a "dapp" that integrates Connext browser node) does not know anything about the iframe's config. It's basically a black box.